### DISCUSSION TITLE
OSError: [Errno 121] Remote I/O error

### DISCUSSION THREAD

**Discussion by federista** [NVIDIA-AI-IOT/jetbot#556] [General]
I am using Waveshare Jetbot with Jetson nano 4GB. After going through all the required steps from [https://github.com/nvidia-ai-iot/jetbot/wiki/software-setup"](url) " I am facing an error in the Basic_motion.ipynb.
The error is:
OSError: [Errno 121] Remote I/O error
I face the error after running executing robot = Robot()
Looking forward to any sort of feedback. Thank you

**Comment by Hexray1108** [comment]
我今晚也遇到了这个问题，解决的方式是重新检查Hardwwre Setup中Step 11 - Wire motor driver to PiOLED display，检查每根线是否真的接对了位置，我本来很笃定我接对了，直到有人指出这个问题，我才发现上面是上3下1，而不是我想象的上2下2的排布。

希望这能帮助你。
